HACKING.rst

```diff
@@ -7,14 +7,14 @@ Next, each time you start working, enter the environment::
 
   source .venv/bin/activate
 
-The HaaS has an unspecified dependency on the ``moc-rest`` library:
-
-.. https://github.com/cci-moc/moc-rest
+The HaaS has an unspecified dependency on the ``moc-rest`` library, which we
+hope to `remove in the future <https://github.com/CCI-MOC/haas/issues/394>`_.
+It may be found at: https://github.com/cci-moc/moc-rest .
 
 The recommended way for developers to satisfy this is to clone that repository,
-and then (while in the virtual environment)::
+and then while in the virtual environment::
 
-  pip install -e ${path_to_moc_rest_clone}
+  pip install 'git+https://github.com/CCI-MOC/moc-rest'
```

Comment:
The drawback to this (and the reason I didn't write it this way in the first place, dispite knowing about this functionality) is that you don't have a checkout of `moc-rest` that you can hack on -- with the old version of the instructions you can easily modify moc-rest when need be.

At this point I don't care too much one way or another; the next change we make to `moc-rest` should be to pull it back into HaaS.